### PR TITLE
Update the Configre Global Settings section

### PR DESCRIPTION
**Pull request by AllForNothing**
1. Add doc for the Skip to update pull time for scanner item

For https://github.com/goharbor/harbor/pull/17807

### CODE REVIEW

**Comment by AllForNothing** [comment]
@OrlinVasilev Kindly help to review the PR as you are more pro in English

**Review by OrlinVasilev — APPROVED** [review]
lgtm

File: docs/administration/general-settings/_index.md [hunk 1]
```diff
@@ -27,3 +27,9 @@ denied: The system is in read only mode. Any modification is prohibited.
 Use the **Project Creation** drop-down menu to set which users can create projects. Select **Everyone** to allow all users to create projects. Select **Admin Only** to allow only users with the Harbor system administrator role to create projects.
 
 ![browse project](../../img/new-proj-create.png)
+
+## Skip updating the pull time for scanner
```
Comment (OrlinVasilev) [inline docs/administration/general-settings/_index.md:31]: Not sure if skip is the right word here, will reach you offline to get some context

Comment (OrlinVasilev) [reply docs/administration/general-settings/_index.md:31]: I think "Retain image last pull time on scanning" is better

**Comment by OrlinVasilev** [comment]
as I mentioned here https://github.com/goharbor/harbor/pull/18431#issuecomment-1486421354 I think "Retain image last pull time on scanning" is clearer

**Comment by AllForNothing** [comment]
@OrlinVasilev PR updated and ready for review

**Review by OrlinVasilev — APPROVED** [review]
fantastic thanks!

**Review by stonezdj — APPROVED** [review]

**Pull request merged**